src/Veldrid/MTL/MTLGraphicsDevice.cs

```diff
@@ -52,15 +54,20 @@ internal unsafe class MTLGraphicsDevice : GraphicsDevice
         {
             _device = MTLDevice.MTLCreateSystemDefaultDevice();
             _deviceName = _device.name;
-            MetalFeatures = new MTLFeatureSupport(_device);
+            MetalFeatures = new MTLFeatureSupport(_device);
+
+            int major = (int)MetalFeatures.MaxFeatureSet / 10000;
+            int minor = (int)MetalFeatures.MaxFeatureSet % 10000;
+            _apiVersion = new GraphicsApiVersion(major, minor, 0, 0);
+
             Features = new GraphicsDeviceFeatures(
                 computeShader: true,
                 geometryShader: false,
                 tessellationShaders: false,
                 multipleViewports: MetalFeatures.IsSupported(MTLFeatureSet.macOS_GPUFamily1_v3),
                 samplerLodBias: false,
-                drawBaseVertex: true,
-                drawBaseInstance: true,
+                drawBaseVertex: true,   // TODO?: MetalFeatures.IsDrawBaseVertexInstanceSupported()
```

Comment:
I think these should be changed, based solely on this table: https://developer.apple.com/metal/Metal-Feature-Set-Tables.pdf

Reply:
I made this change during the push to master.